tools/dockerfile-llb-frontend/test/e2e_util.go

```diff
@@ -0,0 +1,167 @@
+// Package test contains our end to end test suite.
+package test
+
```

Comment:
Can you please add a copyright preamble to the top of this file?
```go
// SPDX-License-Identifier: BSD-3-Clause
// Copyright (c) 2022, Unikraft GmbH and The KraftKit Authors.
// Licensed under the BSD-3-Clause License (the "License").
// You may not use this file except in compliance with the License.

// Package test contains our end to end test suite.
package test
```

Reply:
Done